client/src/components/Common/Button/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, ConfigProvider } from 'antd';
+import 'antd/dist/antd.css';
+
+const ButtonComponent = ({
+  Content,
+  disabled,
+  loading,
+  size,
+  block,
+  onClick,
+}) => (
+  <ConfigProvider direction="rtl">
+    <Button
+      type="primary"
+      disabled={disabled}
+      loading={loading}
+      size={size}
+      block={block}
+      onClick={onClick}
+      className="button-component"
+      style={{ backgroundColor: '#126fc9', borderColor: '#126fc9' }}
```

Comment:
This static stile does not need to, I think if you try to add class and style and import it, we can do any overwrite. 
className= {`come class static ${new className}`}